fdbserver/ResolverInterface.h

```diff
@@ -94,9 +102,21 @@ struct ResolveTransactionBatchReply {
 	// Each group's previous commit version
 	std::map<ptxn::TLogGroupID, Version> previousCommitVersions;
 
+	// Privatized mutations with tags, one for each TLog location
+	VectorRef<StringRef> privateMutations;
+	uint32_t privateMutationCount;
+
 	template <class Archive>
 	void serialize(Archive& ar) {
-		serializer(ar, committed, stateMutations, debugID, conflictingKeyRangeMap, previousCommitVersions, arena);
+		serializer(ar,
+		           committed,
+		           stateMutations,
+		           debugID,
+		           conflictingKeyRangeMap,
+		           previousCommitVersions,
+		           privateMutations,
```

Comment:
shouldn't the newly added field only append to the end of the list?

Reply:
`arena` is different and has to be the last one, and `privateMutations` depends on `arena` here.